generated/ruby/test/errorreporting.rb

```diff
@@ -0,0 +1,165 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
File location is a bit misleading. The test is not generated but hand written. Might want to consider moving the file to tests/ruby/google-cloud-ruby/google-cloud-error_reporting/